src/app-router.js

```diff
@@ -28,7 +28,11 @@ export class AppRouter extends Router {
   reset() {
     super.reset();
     this.maxInstructionCount = 10;
-    this._queue = [];
+    if (!this._queue) {
+      this._queue = [];
```

Comment:
We could refactor it. I was trying to not duplicate the code though. What do you guys think?